docs/videos/video-1-script-draft.txt

```diff
@@ -0,0 +1,23 @@
+Discovering Enterprise Event Bus or What is Event Bus?
+
+Event Bus is an event processing system that offers asynchronous and real-time event transfer between systems within the VA. 
+
+    - What Is An Event?
+
+An event in this context is data describing something that took place in a VA system. Examples of things that could be events include medical claim processing information, contact center ticket information, or Veteran personal data updates. Any data that would be useful to share between systems immediately as soon it's available could be packaged into an event.
```

Comment:
This is an area in our research where we found a lack of understanding about what an event represents. Just giving examples of what an event might be doesn't help to add context. The last sentence of this paragraph does a better job of explaining what an event is than the examples themselves. Maybe switch the last two sentences?

Reply:
Explaining that an event is _data_ , showing a before and after slice of an event data object, and highlighting even a timestamp change and then explaining why that is an event, might be helpful.

Reply:
Planning on adding some visuals to accompany, so I can try and do this as part of the visual portion.